### DISCUSSION TITLE
typo on the issues/flags page

### DISCUSSION THREAD

**Discussion by MartinNohr** [community/community#33291] [Other Feature Feedback, Questions, & Ideas]
The text on the web page is "nice to have festures". I'm pretty sure it should be "features".

**Comment by airtower-luna** [comment]
That does look like a typo, which website do you mean though? :slightly_smiling_face:

**Comment by MartinNohr** [comment]
Example: but you can choose any issues page I suppose since the error is in the drop down list for labels.
https://github.com/MartinNohr/MagicImageWand/issues/62
Click on the labels 'gear' and it is in the drop down list.

**Comment by airtower-luna** [comment]
Oh, you mean the label descriptions? https://github.com/MartinNohr/MagicImageWand/labels As the repository owner you should see "Edit" links on the right that lets you adjust the description.

I'm not sure if that's a GitHub bug: None of my repositories have the "feature request" label, but it could've been added to the defaults later. Or someone made a typo while creating the label in your repository.
